app/utils/validation.ts

```diff
@@ -53,6 +53,16 @@ export const timeIsAfter = (otherField, message) => (value, context) => {
   return [true];
 };
 
+export const isValidAllergy =
+  (
+    message = 'La feltet stå tomt hvis du ikke har noen allergier/preferanser'
+  ) =>
+  (value: string) => {
+    const notValidAnswers = ['ingen', 'ingenting', 'nei', 'nope', 'nada'];
```

Comment:
I believe we should also include "nay", "nah" and "negative" :)

Reply:
Are there any examples of people entering them in the allergies field?